package-lock.json

```diff
@@ -0,0 +1,14207 @@
+{
```

Comment:
I mentioned this in Veronica's PR too - this should be in a separate PR, not this one - it's unrelated.